### DISCUSSION TITLE
How do I remove model save or create from refreshing my app page?

### DISCUSSION THREAD

**Discussion by macroblitz** [sequelize/sequelize#17500] [Help & Questions]
Hello everyone,

I am trying to make my create query (Add user) not to refresh the page whenever I submit the form. I tried everything within the front-end script to no avail. Now, I found out that after commenting the "newRecord.save()" line, my page will not refresh anymore. I do wonder if there's some option or workaround to insert,update,delete records without the page redirect/reload? Or am I doing something wrong here? Thank you.

```
ipcMain.handle('new-user', async (event, data) =>{
       
        const username = data.username
        const password = data.password
        
        try{
            
            const newRecord = user.build({username, password})
            
            newRecord.save()
            return true
        } catch(err) {
            console.log(err)
            return false
        }
    })
```

**Comment by macroblitz** [comment]
I think my module electron-reload is the one doing the refresh thing when my database file is changed.

**Reply by Maus3rSR** [reply]
Did you solved your issue? I have the same. With a simple express server and sqlite.
